wire-ios-sync-engine/Source/Use cases/GetUserClientFingerprintUseCase.swift

```diff
@@ -59,22 +59,19 @@ public class GetUserClientFingerprintUseCase: GetUserClientFingerprintUseCasePro
     public func invoke(userClient: UserClient) async -> Data? {
         let objectId = userClient.objectID
 
-        var existingUser: UserClient?
-        var shouldEstablishSession = false
-        var clientIds = Set<QualifiedClientID>()
-
-        await self.context.perform {
-            existingUser = try? self.context.existingObject(with: objectId) as? UserClient
-            shouldEstablishSession = existingUser?.hasSessionWithSelfClient == false
-            if let id = existingUser?.qualifiedClientID {
-                clientIds.insert(id)
-            }
+        guard let (existingClient, clientId) = await self.context.perform({
+            let client = try? self.context.existingObject(with: objectId) as? UserClient
```

Comment:
optional: Every `try?` might cost additional time for debugging, in my opinion.
Why not logging something instead?

Reply:
We can, I didn't introduce the `try?` I only refactored existing code.